File: projects/10/ruby/Ostrich.rb
# Ostrich.rb | Defines the Ostrich class which inherits attributes and methods
#   from the Bird superclass.
#
# Created by: David Yoo (gy24)
# Date: Mar 29, 2022
####################################################

require './WalkingBird.rb'

class Ostrich < WalkingBird
    def call
        'Snork!'
    end



end

File: projects/10/java/Ostrich.java
/**
 * Ostrich.java provides a ostrich class.
 *
 * Created by: David Yoo (gy24)
 * Date: Mar 29, 2022
 */

public class Ostrich extends WalkingBird {
    /**
     * implicit constructor
     */
    public Ostrich() {
	    super();
    }

    /**
     * explicit constructor
     */
    public Ostrich(String name) {
	    super(name);
    }

    /**
     * Get ostrich's call.
     */
    public String call() {
	    return "Snork!";
    }
}